core/tests/protractor_utils/ExplorationEditorSettingsTab.js

```diff
@@ -65,11 +65,13 @@ var ExplorationEditorSettingsTab = function() {
   this.deleteExploration = async function() {
     await waitFor.elementToBeClickable(
       deleteExplorationButton, 'Delete Exploration button is not clickable');
-    await deleteExplorationButton.click();
+    await action.click('delete Exploration Button', deleteExplorationButton);
```

Comment:
nit: Start with Caps.

```suggestion
    await action.click('Delete Exploration Button', deleteExplorationButton);
```

Reply:
ditto below.